datadog/resource_datadog_downtime.go

```diff
@@ -20,6 +21,80 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+type downtimeOrDowntimeChild interface {
+	GetId() int64
+	GetActive() bool
+	GetDisabled() bool
+	GetMessage() string
+	GetMonitorIdOk() (*int64, bool)
+	GetTimezone() string
+	GetRecurrenceOk() (*datadogV1.DowntimeRecurrence, bool)
+	GetMonitorTags() []string
+	GetStart() int64
+	GetEnd() int64
+	GetScope() []string
+	GetActiveChild() datadogV1.DowntimeChild
+	GetActiveChildOk() (*datadogV1.DowntimeChild, bool)
+}
+
+// downtimeChild wraps the `datadogV1.DowntimeChild` struct via embedding to implement `downtimeOrDowntimeChild`
+// interface missing the `GetActiveChild`, `GetActiveChildOk` methods.
+type downtimeChild struct {
+	child *datadogV1.DowntimeChild
+}
+
+func (d *downtimeChild) GetId() int64 {
```

Comment:
I'm implementing an interface for the open-api-spec generated code, but the CI is failing on `golint` due to preferring `GetID` over `GetId` (breaks the open-api-spec interface). Not sure the best way to ignore this in CI as I'm unfamiliar with Golang.

Reply:
`golint` was deprecated by Google and `NOLINT` was not implemented as a feature before doing so. 

- https://github.com/golang/lint/issues/114
- https://github.com/golang/go/issues/38968